### ISSUE TITLE
Hover to zoom characters in roperty windows"

### ISSUE THREAD

**Issue by eddrda** [HaikuArchives/FontBoy#35]
We can change character size now with the split handler, but it will shrink the character table. Instead

**Comment by humdingerb** [comment]
Is zooming really needed?
Personally, I don't even see the need in the split view. The default width of the left strip of the window is OK (better if the the font name etc weren't truncated). The selected character is nice and big and making it huge by reducing the list of all characters to the right doesn't make much sense to me...

**Comment by eddrda** [comment]
I think in most cases, it isn't necessary. Though fonts do contain some faults, and only with zooming can we detect them. Here's an example (inconsolata, Google Fonts):
![untitled-1](https://user-images.githubusercontent.com/19305859/34468088-f411d37e-ef33-11e7-99fd-3c7cc380ee46.png)

**Comment by humdingerb** [comment]
It's probably not the usual use-case for FontBoy, but OK...
The split view is still the wrong means for zooming. Maybe the mouse wheel on the preview could zoom in/out where the mouse pointer is.

**Comment by janus2** [comment]
I think FontBoy more as a catalog to consult the installed fonts... I don't thinks need to advanced functionality... I don't think it should be used to inspect glitches in fonts.

**Comment by humdingerb** [comment]
No, but zooming in/out of that preview would be a nice intuitive feature that doesn't need additional widgets to clutter the GUI. If anyone wants to implement that it's OK. Otherwise the issue just sits in this tracker... :)
